llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -1064,6 +1064,8 @@ class LLVM_ABI_FOR_TEST VPInstruction : public VPRecipeWithIRFlags,
     ResumeForEpilogue,
     /// Returns the value for vscale.
     VScale,
+    /// Extracts all lanes from its (non-scalable) vector operand.
```

Comment:
This is an abstract VPInstruction whose single defined VPValue represents VF scalars extracted from a vector, to be replaced by VF ExtractElement VPInstructions?

Reply:
Added thanks